src/features/connections/services/connectionsApiSlice.ts

```diff
@@ -1,2 +1,50 @@
-// boiler plate
-import React from "react";
\ No newline at end of file
+import {
+  ConnectionsModel,
+  ConnectionMutationResponseModel,
+} from "../models/Connections.model";
+import { apiSlice } from "../../api/apiSlice";
+
+export const connectionsApiSlice = apiSlice.injectEndpoints({
+  endpoints: (builder) => ({
+    getConnections: builder.query<ConnectionsModel, number | string>({
+      query: (userId: string | number) => ({
+        url: `/users/${userId}/connections`,
+        method: "GET",
+      }),
+    }),
+    sendConnection: builder.mutation<
+      ConnectionMutationResponseModel,
+      number | string
```

Comment:
i would avoid having mixed types here. Type the userId to whatever the data type is when we receive it from the backend